src/main/resources/static/css/main.css

```
@@ -820,6 +825,280 @@ button:disabled {
+    outline: 2px solid var(--color-accent);
+    outline-offset: 2px;
+}
+
+.connections-empty-state {
+    padding: 12px 4px;
+    color: var(--color-text-muted);
+    font-size: 0.82rem;
+}
+
+
+/* Connections to messages button*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)